server/application/application.proto

```diff
@@ -18,6 +18,7 @@ import "github.com/argoproj/argo-cd/reposerver/repository/repository.proto";
 message ApplicationQuery {
 	optional string name = 1;
 	required bool refresh = 2 [(gogoproto.nullable) = false];
+	optional string project = 3;
```

Comment:
can we make project be: [(gogoproto.nullable) = false] ?

Reply:
I've realized I need to be able to filter by multiple projects. Changed